jujugui/static/gui/src/app/components/model-actions/model-actions.js

```diff
@@ -83,6 +84,12 @@ class ModelActions extends React.Component {
         </span>
       );
     }
+    let terminalAction = null;
+    if (false) {
```

Comment:
To allow for testing and for this type of flagging maybe pass a prop in 'terminalEnabled' which can then be passed in the tests so it can be tested properly

Reply:
This will be done in a follow-up, we have a card for turning this stuff a proper feature flag.